### ISSUE TITLE
Request failed with status code 403

### ISSUE THREAD

**Issue by jscheeres** [piraces/azure-devops-bicep-task#24]
Bicep install often fails with a 403 from Azure DevOps pipeline:

```
Starting: BicepInstall
==============================================================================
Task         : Install Bicep CLI
Description  : Install Bicep CLI
Version      : 0.3.8
Author       : Raúl Piracés
Help         : Installs the Bicep CLI
==============================================================================
##[error][FATAL] Error while retrieving latest version tag: 'Request failed with status code 403'
Finishing: BicepInstall
```

**Comment by piraces** [comment]
Hi @jscheeres ! Thanks for reporting. I will review it as soon as I can. It is always the same log error, while retrieving latest version tag?
Maybe we are reaching a rate-limit or some kind of restricition in GitHub... Will investigate 👍

**Comment by jscheeres** [comment]
Yes always the same error. We experience it almost every day during a release. The problem is solved after executing the pipeline again.

**Comment by piraces** [comment]
Hi @jscheeres , I have been trying making multiple runs and I have been able to reproduce the issue in some runs… It appears that the issue is from the GitHub API, that now requires a `User-Agent` headers for all requests: https://docs.github.com/en/rest/overview/resources-in-the-rest-api?apiVersion=2022-11-28#user-agent-required

This has been fixed and released to the store (v4.0.1). Please try it out and let me know if this keeps happening to you.
Feel free to reopen the issue and comment if you find the same issue.

**Comment by piraces** [comment]
Thanks for reporting this, I think I made an error while publishing the new versions and changes are not being applied... I'll let you know when fixed.

**Comment by piraces** [comment]
Hi @jscheeres , I just released v0.4.3 with this fix... Can you try it to reproduce it again? Thank you and sorry for the confusion 😞

**Comment by jscheeres** [comment]
Great. We will try it. Thanx.

**Comment by piraces** [comment]
Hi @jscheeres , just reproduced again with the latest version... will need to investigate further...

**Comment by jscheeres** [comment]
Starting: Bicep Install
Task         : Install Bicep CLI
Description  : Install Bicep CLI
Version      : 0.4.4
Author       : Raúl Piracés
Help         : Installs the Bicep CLI
##[error][FATAL] Error while retrieving latest version tag: 'Converting circular structure to JSON'.

**Comment by piraces** [comment]
Should be fixed by now @jscheeres , I have completely changed the way to obtain the latest version tag. Instead of using the GitHub API, I'm now using URL redirections. Please confirm if its solved on your side with the v0.4.5 🙏
